core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1204,7 +1258,8 @@ public ManagedChannel createOobChannel(EquivalentAddressGroup addressGroup, Stri
               "OobChannel for " + addressGroup);
       final OobChannel oobChannel = new OobChannel(
           authority, balancerRpcExecutorPool, transportFactory.getScheduledExecutorService(),
-          syncContext, callTracerFactory.create(), oobChannelTracer, channelz, timeProvider);
+          syncContext, callTracerFactory.create(), oobChannelTracer, channelz, timeProvider,
+          configSelector);
```

Comment:
I although thought about it, but I also noticed that `OobChannel` uses the same load balancing policy as the parent channel. So it should use the same config selector, otherwise that same lb policy may not work. What should not have the same service config with the parent channel is the oob channel from `Helper.createResolvingOobChannel()` which does its own name resolution.

Reply:
Fixed